src/bindgen/ir/function.rs

```diff
@@ -284,6 +293,21 @@ impl Source for Function {
                         out.new_line();
                     }
                 }
+                if let Some(note) = func.annotations.deprecated_note(config) {
+                    if note.is_empty() {
+                        if let Some(ref anno) = config.function.deprecated {
+                            write!(out, "{}", anno);
+                            out.new_line();
+                        }
+                    } else if let Some(ref anno) = config.function.deprecated_with_note {
```

Comment:
Any reason this is not using the helper?

Reply:
No, made it use the helper.